blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.scss

```diff
@@ -8,37 +8,70 @@ body {
   display: flex;
   flex-direction: column;
   min-height: 100vh;
+}
 
-  .page-header {
-    position: sticky;
-    top: 0;
-    width: 100%;
-    z-index: 9;
-  }
+.page-header {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  z-index: 9;
+}
 
-  .main {
-    flex-grow: 1;
-    transition: padding-top 0.3s linear;
-    width: 100%;
+.main {
+  flex-grow: 1;
+  transition: padding-top 0.3s linear;
+  width: 100%;
 
 
-    @media screen and (max-width: 95rem) and (min-width: 90rem) {
-      .container {
-        margin-left: 5%;
-        margin-right: 5%;
-      }
+  @media screen and (max-width: 95rem) and (min-width: 90rem) {
+    .container {
+      margin-left: 5%;
+      margin-right: 5%;
     }
   }
+}
 
-  .left-article-section {
-    padding: 0;
-    @media screen and (min-width: map-get($grid-breakpoints, 'lg')) {
-      border-right: 1px solid $border-rule-color;
-      padding: 0 map-get($spacers, 'lg') 0 0;
-    }
+.advanced-grid-desktop-main-area {
+  padding: 0;
+  @media screen and (min-width: map-get($grid-breakpoints, 'lg')) {
+    border-right: 1px solid $border-rule-color;
+    padding: 0 map-get($spacers, 'lg') 0 0;
+  }
+}
+
+
+@media (max-width: 64rem) {
```

Comment:
Was this tested under IE 11?.....I believe it's still supported.  Need to make sure it falls back gracefully.....